testpar/t_subfiling_vfd.c

```diff
@@ -2414,7 +2487,30 @@ main(int argc, char **argv)
     if (num_iocs_g > mpi_size)
         num_iocs_g = mpi_size;
 
+#ifdef H5_HAVE_FILTER_DEFLATE
+    if (MAINPROCESS) {
+        printf(" Re-running tests with compression enabled\n");
+    }
+    enable_compression = true;
+    for (size_t i = 5; i < ARRAY_SIZE(tests); i++) {
```

Comment:
The first few tests do everything the same when compression is enabled.

Reply:
It might be useless to run the tests again, but from a maintenance perspective it would make sure that tests don't get missed in the future. We could also separate the tests into different arrays, but that's not very scalable either if we want to test another feature in the future.

Reply:
I can add a comment and a variable instead of hard coding a number.

Reply:
It's probably easiest to just re-run everything again, since someone could also add a non-compression-related test in the middle of the current set of tests and it would get run anyway.